### PR TITLE
DA archive node

### LINKED ISSUE #145: Add dependencies for archive relay to opam.export
Will also unblock #119.

`websocket-async` is missing.

### PR DESCRIPTION
**Pull request by MartinOndejka**
Changes the graphql API so that the archive relay, fetches the current ledger hash from sequencer and syncs the state of archive node with the DA node.

Closes: #145

### CODE REVIEW

File: opam.export [hunk 1]
```diff
@@ -1,15 +1,18 @@
 opam-version: "2.0"
-compiler: ["ocaml.4.14.0"]
```
Comment (L-as) [inline opam.export]: This file needs to be cleaned up with git reset -p.

File: nix/ocaml.nix [hunk 2]
```diff
@@ -265,6 +266,7 @@ let
           cp src/app/zeko/sequencer/run.exe $zeko/bin/zeko-run
           cp src/app/zeko/sequencer/deploy.exe $zeko/bin/zeko-deploy
           cp src/app/zeko/sequencer/cli.exe $zeko/bin/zeko-cli
+          cp src/app/zeko/sequencer/archive_relay/run.exe $zeko/bin/zeko-archive-relay
```
Comment (L-as) [inline nix/ocaml.nix]: There should be a separate output called zeko_archive_relay.

File: opam.export (hunk 1 above)
Comment (MartinOndejka) [reply opam.export]: I've removed some of them, I'm not sure what exactly should be removed though.

File: opam.export [hunk 3]
```diff
@@ -178,8 +191,10 @@ installed: [
   "ocaml-syntax-shims.1.0.0"
   "ocaml-version.3.4.0"
   "ocamlbuild.0.14.0"
+  "ocamlc-loc.3.5.0"
   "ocamlfind.1.9.3"
   "ocamlformat.0.20.1"
+  "ocamlformat-rpc-lib.0.26.1"
```
Comment (L-as) [inline opam.export:196]: Not needed technically but ocaml lsp always complains that it can't find it so OK I suppose.

File: opam.export [hunk 4]
```diff
@@ -160,6 +172,7 @@ installed: [
   "menhirSdk.20210419"
   "merlin.4.5-414"
   "merlin-extend.0.6.1"
+  "merlin-lib.4.12-414"
```
Comment (L-as) [inline opam.export]: Quite possibly not needed

**Review by L-as — COMMENTED** [review]
Did you check if `nix develop .?submodules=1#with-lsp` still works?

**Comment by MartinOndejka** [comment]
> Did you check if `nix develop .?submodules=1#with-lsp` still works?

It does

**Review by L-as — APPROVED** [review]
LGTM

**Pull request merged**